DXRVanilla/DeusEx/Classes/PersonaScreenInventory.uc

```diff
@@ -179,10 +179,17 @@ function EnableButtons()
         inv = Inventory(selectedItem.GetClientObject());
 
         if (inv != None) {
-            if (class'DXRLoadouts'.static.IsRefused(inv.class)) {
-                btnRefusal.SetButtonText(AcceptLabel);
-            } else {
-                btnRefusal.SetButtonText(RefuseLabel);
+            switch (class'DXRLoadouts'.static.GetLootAction(inv.class)) {
+                case 0:
+                    btnRefusal.SetButtonText(RefuseLabel);
+                    break;
+                case 1:
+                    btnRefusal.SetButtonText(AcceptLabel);
+                    break;
+                case 2:
+                    btnRefusal.SetButtonText(RefuseLabel);
+                    btnRefusal.DisableWindow();
```

Comment:
soy food isn't disabling the trash button for me

![image](https://github.com/user-attachments/assets/6139ba0c-e48d-43f3-bf2c-bae58562a301)

![image](https://github.com/user-attachments/assets/88e7def7-8a5b-4639-a42c-c4d0339a0eb9)

crowbar is correctly updating the button

![image](https://github.com/user-attachments/assets/fb2f5a53-fbd4-4680-bfa2-e19754af80f3)

![image](https://github.com/user-attachments/assets/c7205b7e-225d-4016-a67b-9836d6dafa9a)

Reply:
disregard this one, I was still a commit behind